bin/set-up-current-account.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+set -euo pipefail
+
+# Name the account based on the current account alias
+ACCOUNT_ALIAS="$(./bin/current-account-alias.sh)"
```

Comment:
Something to consider, I acquired a fresh account that did not have an alias setup initially, of course this broke this script until I setup that alias. It may be good to add a warning that an alias is needed or if an alias is not found to use the account id instead.

Reply:
Thanks! I was actually wondering if all accounts have an alias.

What do you think we should do. I like the idea of being robust and just using account id, but from a developer experience perspective it's really nice to be able to see a folder with names like `project-dev.tfbackend, project-prod.tfbackend` rather than `123415345.tfbackend`, `23452362.tfbackend`.

If we fallback to id, then after the account is configured, if an alias is added, they'd need to rename the file to the alias otherwise it won't work.

This makes me wonder:
1. Can an account have two or more aliases?
2. Can an account change their alias?

What are your thoughts?

Reply:
1. An account can have only a single alias
2. As the root user of my account I was able to set the alias. I also found out recently a fresh bid account was actually created for mcads but fell through the cracks somehow. In that account I updated the alias from `bid-fed-mcads-tech-challenge` to `bid-tech-challenge` and transferred that account to Haiyan, so it is possible. There is also a miss named account alias in my projects greenfield project that will need to be updated. 

I agree that `prod.tfbackend` is easier on the brain than `123456789012.tfbackend`. I am just not sure how to make it easy to read and guarantee its correct.  I have been going back and forth between the following:
1. Requiring template users to ensure they have an account alias and provide instructions for if it changes.
2. Just using the account_id since once setup is complete there wont be a lot of changes happening. 
3. Allowing the user to name their own backend file and refactoring for that. 
4. Combine 2 & 3 using some kind of “symlink” to link them or including `myname.account_id.s3.tfbackend`.

Reply:
I have an idea, for accounts, how about when we create the backend config file in `set-up-current-account.sh`, we name the file `<alias>.<account-id>.tfbackend`, where `<alias>` is a user specified alias. It can be the account alias, or whatever, but it's not actually used. Perhaps it can default to the account alias if it exists. When running things, in order to find the right tfbackend file, we can do something like `ls | grep <account-id>`.

Since running things only uses the account-id, the alias is purely for readability when looking at the file system, so someone can rename the file if they want to whatever is most readable.

Reply:
I went ahead and implemented that last idea. Take a look and lemme know what you think!

Reply:
That's good I like it.